recipes/markdown-word-count/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "markdown-word-count" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/markdown_word_count-{{ version }}.tar.gz
+  sha256: 56095ce1b1934a988b8dbe7dd5d9bb3f3ffcf353afa3f46adc8c26ea56501d31
+
+build:
+  entry_points:
+    - mwc=mwc.cli:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - mwc
+  commands:
+    - pip check
+    #- mwc --help
```

Comment:
Why is this commented out?

Reply:
This app has no help message.  It requires a markdown file to provide a zero status output, however none is shipped with the package.  I could simulate one, but I think this would be beyond the purpose of the package?

Reply:
Thanks for this review.  I implemented a simple test as requested on the last push.

Reply:
The test does not work because the program sets the exit code `!= 0` always, despite the fact it works...  I'll check it later today.

Reply:
This is now fixed, and ready for review @zklaus